cmd/seed-controller-manager/options.go

```diff
@@ -140,7 +140,7 @@ func newControllerRunOptions() (controllerRunOptions, error) {
 	flag.StringVar(&c.oidcIssuerClientID, "oidc-issuer-client-id", "", "Issuer client ID")
 	flag.StringVar(&c.oidcIssuerClientSecret, "oidc-issuer-client-secret", "", "OpenID client secret")
 	flag.StringVar(&c.kubermaticImage, "kubermatic-image", resources.DefaultKubermaticImage, "The location from which to pull the Kubermatic image")
-	flag.StringVar(&c.etcdLauncherImage, "etcd-launcher-image", resources.DefaultEtcdLauncherImage, "The location from which to pull the etcd launcher image")
+	flag.StringVar(&c.etcdLauncherImage, "etcd-launcher-image", resources.DefaultEtcdLauncherImage, "The location from which to pull the etcd launcher image, if image tag is not corresponding dynamically derived kubermatic commit is used")
```

Comment:
Nit pick, but can we make the phrasing a bit easier to parse, like

> to pull the etcd-launcher; if image tag is not corresponding [to what?], *then* a dynamically derived Kubermsatic commit is used [which one?]

The seed-controller should have access to the version/git commit it was built for, maybe it could be nice to just fmt.Sprintf() the actual docker image tag that would be used. like

```
fmt.Sprintf("Do that, if not then the image tag %s is used", versions.KUBERMATICDOCKERTAG) // pseudo code
```